benchmarks/scripts/expval_benchmark.py

```diff
@@ -158,35 +159,91 @@ def fetch_pre_post_compiled_circuits(
     return uncompiled_qiskit_circuit, compiled_qiskit_circuit
 
 
+def get_heavy_bitstrings(circuit: qiskit.QuantumCircuit) -> Set[str]:
+    """ "Determine the heavy bitstrings of the circuit."""
+    simulator = AerSimulator(method="statevector")
+    result = simulator.run(circuit).result()
+    counts = list(result.get_counts().items())
+    median = np.median([c for (_, c) in counts])
+    return set(bitstring for (bitstring, p) in counts if p > median)
+
+
+def estimate_heavy_output(
+    circuit: qiskit.QuantumCircuit,
+    qv_1q_err: float = 0.002,
```

Comment:
For posterity sake, how would we know how these values were determined? Should we have some comments or details on that process, in the event we need to revisit or there's a question?

In hindsight, I have the same question for @natestemen and the other observable benchmark he added.

Reply:
> how would we know how these values were determined?

I played with "reasonable" valuables. Meaning values that are within the range of what we are seeing on hardware. There wasn't much more than this.